lib/sepia/src/util.js

```diff
@@ -129,7 +129,10 @@ function substituteWithOpaqueKeys(text) {
   var substitutions = globalOptions.substitutions;
   for (var i=0; i<substitutions.length; i++) {
     var subst = substitutions[i];
-    text = text.replace(subst.actualValueFn(), subst.opaqueKey);
+    var actualValue = subst.actualValueFn();
```

Comment:
Thanks!  Yeah I noticed that as well and had not had time to fix it.  I think this would be faster than the loop
text = text.replace(new RegExp(subst.actualValueFn(), 'g'), subst.opaqueKey);